data/cncf.yaml

```diff
@@ -237,6 +267,7 @@
       check_sets:
         - community
         - code
+        - docs
```

Comment:
The `docs` check set is a subset of the `code` check set, so it isn't needed in this case. Please remove it.